docs/norns/faq.md

```diff
@@ -0,0 +1,84 @@
+# FAQ
+
+## what is the difference between `require` and `include`?
+
+- `require()` is a built in lua function, `include()` is a norns function
+
+- `require()` looks in more places - see `norns/lua/core/config.lua`
+
+- most importantly, `require()` caches its results, and will not re-run its arguments (unless you use a hack[^1]) the state of a `require`'d module is persisted, and this is significant. consider the following set of files. 
```

Comment:
perhaps move the paren?

=>

most importantly, `require()` caches its results, and will not re-run its arguments (unless you use a hack[^1] the state of a `require`'d module is persisted), and this is significant.